app/views/home/_header.html.erb

```diff
@@ -12,11 +12,11 @@
           </li>
         <% else %>
           <li class="nav-item">
-            <%= link_to 'Log In', user_session_path, class:"nav-link-lg"  %>
-          </li>
+            <%= link_to 'Get Started', user_session_path, class:"nav-link-lg"  %>
+          </li> 
```

Comment:
Maam might ask us to define a sign up and log in especially if the user is already a member. Get started only applies to applying users. Do we need this change?